docs/data-guidelines/index.md

```diff
@@ -52,6 +52,30 @@ Do not use `"preview"` for planned but not yet implemented support changes. In o
 
 This guideline was adopted to protect the quality of stable data in the face of schedule uncertainty. To learn more about the adoption of `"preview"` values, see [#12344](https://github.com/mdn/browser-compat-data/issues/12344) and [#10334](https://github.com/mdn/browser-compat-data/pull/10334).
 
+## `"partial_implementation"` general usage guidelines
+
+You must set `"partial_implementation": true` when all of the following conditions are met:
+
+- The browser's support does not implement mandatory specified behavior.
+- The browser's support is inconsistent with at least one other browser.
+- The browser's support misleads or evades feature detection.
+
+Here are some example situations:
+
+- `"partial_implementation": false`: All implementing browsers ignore part of a feature's specified behavior in the same way.
+  This behavior is consistent and is a _de facto_ complete implementation.
+
+- `"partial_implementation": false`: All implementing browsers provide a form control user interface, but the specification gives the implementer discretion over its look and feel.
+  A developer complains that one browser's user interface lacks some desired quality that other browsers implement; they want it to be marked as partially implemented.
+  Use a note or non-standard behavioral subfeature instead.
+
+- `"partial_implementation": true`: `CSS.supports()` returns `true` for a property name and value, but the value has no behavior.
+  See [Non-functional defined names imply `"partial_implementation"`](#non-functional-defined-names-imply-partial_implementation).
+
+- `"partial_implementation": true`: One browser implements `ExampleInterface` but exposes it on the `Node` interface instead of the specified `Element` interface.
+  Other implementers use the correct parent interface.
+  This misleads feature detection (`"ExampleInterface" in Element.prototype` → false), so it's partially implemented.
```

Comment:
Nit: The word "misleads" keeps confusing me here. Would this be right:

```suggestion
  This fails feature detection (`"ExampleInterface" in Element.prototype` → false), so it's partially implemented.
```

Reply:
See my previous comment about the interpretation of "failure" in this setting.

We might make this more obvious with a different example though. Perhaps something like:

> `"partial_implementation": true`: One browser exposes a method, `example()`, but due to a bug it's a no-op. Other browsers implement the method's intended behavior. This confuses feature detection because `"example" in ExpectedInterface.prototype` returns `true`, even though the method does not work.

Reply:
Yes, I like that a lot, although I would suggest replacing "it's a no-op" with something like "it has no observable behavior or effect" to make it clearer.

Reply:
I did this, somewhat modified, in 16aab1545ac411d42b683deae50db657f909ff10. It's actually a real example with the names changed: [`Notification()` in Chrome for Android](https://developer.mozilla.org/en-US/docs/Web/API/Notification/Notification#browser_compatibility).